### DISCUSSION TITLE
Call for translations Freeplane 1.11

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#986] [Announcements]
Changes in version 1.11 are open for translations.
Please contribute at https://hosted.weblate.org/projects/freeplane/

**Comment by ldmpub** [comment]
Done for french 😊.

**Reply by bepolymathe** [reply]
Trop rapide ;-)

**Comment by PredragC** [comment]
Done for Croatian.

**Comment by marco64io** [comment]
Revised Italian translation...
